tests/functional/event_handler/required_dependencies/test_api_gateway.py

```diff
@@ -331,7 +331,7 @@ def test_cors():
     def with_cors() -> Response:
```

Comment:
I'm wondering if we can remove `cors=True` here. It's the default behavior when cors is set in `ApiGatewayResolver`. What do you think?